VideoLocker/src/androidTest/java/org/edx/mobile/test/http/SegmentTests.java

```diff
@@ -25,10 +28,13 @@ private void initTracker() {
         if (segment == null) {
             // create mocked instance of SegmentTracker
             tracker = Mockito.mock(ISegmentTracker.class);
+            //The issue with the implemenation of Singleton for ISegment is that
+            //some times the singleton is a dummy object. which causes
+            // the Mocito failed here.
```

Comment:
typo: "Mockito" not "Mocito". Might be worth marking ISegment with a TODO explaining how we want to change it.